golem/core/optimisers/adaptive/utils.py

```diff
@@ -0,0 +1,13 @@
+from functools import partial
+from typing import Callable
+
+
+def get_callable_name(action: Callable):
```

Comment:
nit: совсем мелочи, но нужно бы показать чистую строгую типизацию

```suggestion
def get_callable_name(action: Union[Callable, str, partial]) -> str:
```

Reply:
done